cinema/urls.py

```diff
@@ -11,13 +11,18 @@
 )
 
 router = routers.DefaultRouter()
-router.register("genres", GenreViewSet)
-router.register("actors", ActorViewSet)
-router.register("cinema_halls", CinemaHallViewSet)
-router.register("movies", MovieViewSet)
+
 router.register("movie_sessions", MovieSessionViewSet)
```

Comment:
Only the 'movie_sessions' endpoint is registered with the router. If other viewsets like 'GenreViewSet', 'ActorViewSet', etc., are intended to be used as viewsets, they should also be registered with the router. Otherwise, consider removing the router if it's not needed.